migration/identity-migration/src/main/java/io/camunda/migration/identity/config/ZeebeProperties.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.config;
+
+public class ZeebeProperties {
+
+  private String brokerContactPoint;
```

Comment:
:wrench: Since this is an entirely new configuration (under `camunda.migration.identity.zeebe`) I think we can ignore the backwards compatibility stuff around `brokerContactPoint` and only use `gatewayAddress` directly. So `ZeebeConnector#getGatewayAddress` can be removed too.

Reply:
I'll fix it in the next PR that will be ready by today.